ssr/.gitignore

```diff
@@ -1 +1,2 @@
 include.ts
+!*.js
```

Comment:
I think it's possible we might want to add more inline scripts, it would be weird/annoying to have to add those to .gitignore before checking them in - I may just remove the ignore from the toplevel .gitignore